pages/posts.tsx

```diff
@@ -1,3 +1,5 @@
+import React from "react";
+
 import { useState, useEffect, useRef } from "react";
 import { inter, roboto_serif } from "../styles/fonts";
```

Comment:
Shouldn't all your font initialisations be inside your root layout file?

Or are you saying that it is only this page that would make use of the `Inter` and `Roboto` fonts?